client/my-sites/plugins/hooks/use-get-dialog-text/index.ts

```diff
@@ -47,11 +47,13 @@ const getAffectedSites = ( plugins: Plugin[], sites: Site[] ) => {
 			.map( ( id ) => parseInt( id ) )
 	);
 
-	return (
-		sites
-			// We can't affect any sites that don't allow updating files
-			.filter( ( { canUpdateFiles } ) => canUpdateFiles )
-			.filter( ( s ) => pluginsInstalledOnSiteIds.has( s.ID ) )
+	return sites.filter(
+		( s ) =>
+			( s.canUpdateFiles ||
+				( s.is_a4a_client &&
```

Comment:
Is it guaranteed that `is_a4a_client` will be always available?

Reply:
No, I don't think so. But it shouldn't matter as the `s` is always available here.

However, I made a small refactor: https://github.com/Automattic/wp-calypso/pull/98789/commits/99c6780ec55e44f96a0029afa3c65e6e3ce6eff0